lua/core/nvimtree.lua

```diff
@@ -57,16 +57,12 @@ function M.setup()
   end
   local g = vim.g
 
-  for opt, val in pairs(lvim.builtin.nvimtree) do
```

Comment:
Icons aren't part of the changes introduced into nvim-tree, so removing this causes the icon customizations done above to fail

Reply:
yes 😅